src/simulators/statevector/statevector_controller.hpp

```diff
@@ -122,7 +125,8 @@ void StatevectorController::clear_config() {
   initial_state_ = cvector_t();
 }
 
-size_t StatevectorController::required_memory_mb(const Circuit& circ) const {
+size_t StatevectorController::required_memory_mb(const Circuit& circ,
+                                                 const Noise::NoiseModel& noise) const {
```

Comment:
Why do we need `noise`  here? Seems like we are not using it at all.

Reply:
It's used in the qasm controller because the required memory depends on simulation method (which depends on noise model). There should be a more elegant way of doing this but it would require some more serious refactoring of controller classes that I would like to postpone till later